sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2694,7 +2694,7 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
     sparkContext.addSparkListener(listener)
     try {
       // Execute the command.
-      sql("show databases").head()
+      sql("EXPLAIN show databases").head()
```

Comment:
Why did this change?

Reply:
This is because the new `*Exec` classes extends `LeafExecNode`, which runs a job when `executeCollect` is called. If we need to keep the behavior same (#17027), we can reuse `ExecutedCommandExec` along with `RunnableCommand` or introduce new physical operator for these commands that don't require running a job. @cloud-fan, looks like you reviewed #17027 back in 2017. :) This is still a hard requirement right?

Reply:
Yes it's still a requirement, and seems we break it in all v2 commands. We should have a base class for v2 command physical plan and avoid submitting spark job for it.

Reply:
OK. I will start this on a separate PR.

Reply:
Adressed by #26048